### ISSUE TITLE
Create LichessApi.Web Nuget

### ISSUE THREAD

**Issue by rzakanych** [tanczosm/LichessApi#21]
When do you plan to create nuget?

**Comment by tanczosm** [comment]
Good idea.  I'll try to see if I can get this published.  I can update to net6 as well   This API client could be really good but I'll definitely need some assistance spotting issues.  I authored the LiChess OAuth extension as well so I'm using the library in conjunction with LiChess for authentication.
